database-service/tests/test_migrations.py

```diff
@@ -1,16 +1,57 @@
-import os
-import sh
+# TODO: we need to use the incremental feature of pytest that guarantees that the following tests are run in this order:
 
 
-def test_migrations_can_be_applied_without_errors(hasura_endpoint):
-    # Given I am in the right location
-    assert os.path.exists('migrations')
-    assert os.path.isfile('config.yaml')
+def test_shopozor_structural_migrations_can_be_applied(hasura_client, app_root_folder):
+    # Given I've structural project migrations
+    project_folder = app_root_folder
 
     # When I apply the migrations
-    cmd = sh.Command('hasura')
-    result = cmd('migrate', 'apply', '--endpoint',
-                 hasura_endpoint, '--skip-update-check')
+    result = hasura_client.apply_migrations(project_folder)
 
     # Then I get no errors
     assert 0 == result.exit_code
```

Comment:
To clean up after that test, we would need to roll back the modifications, or to use the postgres adapter to empty the database. I think this is not necessary and we can use the pytest feature allowing to run those tests in a predefined order.

Reply:
fixed